data/BThomas22tech.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "Brandon Thomas",
+  "type": "personal",
+ 
+  "bio": "Full Stack Python developer looking to change the world!"
+  "links": [
+  {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Brandon4Tenn",
+      "icon": "FaTwitter"
+},
+{
+      "name": "Follow me on Github",
+      "url": "https://github.com/BThomas22tech",
+      "icon": "FaGithub"
+},
```

```suggestion
}
```
Please remove the trailing comma